src/Microsoft.PowerShell.ConsoleHost/host/msh/ConsoleHostUserInterface.cs

```diff
@@ -1921,8 +1910,6 @@ internal string ReadLineWithTabCompletion(Executor exec)
                     {
                         lastCompletion = completedInput;
                     }
-
-                    lastInput = completedInput;
```

Comment:
Does tab completion still work when PSReadLine isn't loaded?

Reply:
Yes, it does not work. I will look more closely.
I see and wonder that tab completion doesn't work on Unix without PSReadline at all.

Reply:
@iSazonov Unix has a very simple readline implementation while Windows still enables the tab-complete experience (using Windows APIs for rendering).  Want to make sure we don't introduce a regression here.

Reply:
On Windows, the cooked input mode supports ending on <kbd>Tab</kbd> to support tab completion in a cli like PowerShell or cmd, see [here](https://github.com/PowerShell/PowerShell/blob/0f7efddd22a4a44f61a2f22de86f5d6b6aa47564/src/Microsoft.PowerShell.ConsoleHost/host/msh/ConsoleControl.cs#L753).

On Unix, cooked mode does not end on <kbd>Tab</kbd> and a cli typically ends up in raw mode if it wants tab completion.